test/integration/account/activities_test.rb

```diff
@@ -1,6 +1,6 @@
 require "test_helper"
 
-class AccountActivitiesTest < ActionDispatch::IntegrationTest
+class Account::ActivitiesTest < ActionDispatch::IntegrationTest
```

Comment:
Yeah, let's chat about it later, but my thoughts were that we should aim to refactor things at some point, however:
- the account and users divide runs through much of the app, so it would take quite a substantial amount of time to refactor it all: probably more of a medium-term goal?
- the integration tests are a little chaotic as it stands, and it's hard to know which of them relate to account/applications/users/other stuff. Making what we already have more easily identifiable will help us compare differences when it comes to refactoring out the divide later, should we choose to do that

So this is more of a descriptive change based on the current situation than an indication of our preferred direction of travel, and I think the extra clarity will make it easier to wade through later on

I do agree though that the distinction between the controller and integration tests is a little murky and due for refinement, but again I wonder if it's something for later

Reply:
Fair, it probably is an improvement. I'm happy for us to go ahead with this for now, at least.